src/app.js

```diff
@@ -25,11 +33,64 @@ const quoteData = [
   },
 ];
 
+quoteData.forEach(function(quote) {
+  $('#dropdown').append(`<option>${quote.symbol}</option>`)
```

Comment:
This code should really be within the `$(document).ready(...)` callback function.

The reason for that is because we're using jQuery to find the `select` tag on the page with ID `dropdown`, and then manipulating it by appending `option` tags inside of it. Both of these things are only guaranteed to work properly if we wait until the entire page has been loaded -- which is what the `$(document).ready(...)` is for.

We can see the problem manifest if we change `index.html` to have the `script` tag for `app.bundle.js` inside of the `head` tag after the `meta` tags, instead of at the bottom of the `body` tag. By moving the JS code to the top of our HTML file, the browser will load it and run the JS code in `app.js` first, _before_, loading the actual HTML content of the page.

In that case the drop down selection does not have any options added to it, because when this line of code runs jQuery cannot find the `select` tag with ID `dropdown` (because it hasn't been loaded yet from the HTML file).